### PR TITLE
Update e2e scipts according to new structure

### LINKED ISSUE #420: Integration Tests not Running against new PRs
# Expected Behavior

While raising a new PR, integration tests should run and then should give the result whether passed or failed.

# Actual Behavior

currently all tests are passing because of this new catalog reorganisation. example integration test logs for PR #391 -> [logs](https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_catalog/391/pull-tekton-catalog-integration-tests/1282941809437184002/?log#log) 
#352 -> [logs](https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_catalog/352/pull-tekton-catalog-integration-tests/1283330369541640192/?log#log)

# Steps to Reproduce the Problem

1. Raise a new PR to add a task along with corresponding tests according to the latest catalog structure
2. View the integration test logs

# Additional Info
 This is happening because the catalog is re-organised and structure of tasks have also changed

### PR DESCRIPTION
**Pull request by piyush-garg**
This will update the e2e scripts to run tests according to
new catalog structure after reorg

Fix git batch merge tests

Fix git clone tests

Fix git cli tests

Remove git-rebase tests as they are not of rebase

Fix golnag build tests

Fix golang-test tests

Fix golangci-lint tests

Fix helm-upgrade-from-repo tests

Fix helm-upgrade-from-source tests

Update skopeo task for tls-verify flag param
This will fix skopeo tasks to have param for tls-verify flag
and update tests accordingly

Skip cloudevent task as it is failing

Fix warning for yaml.Load function

Fix upload-pypi pre-apply-task script

Remove python pypi upload test

Fix #420 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Yaml file complies with [yamllint](https://github.com/adrienverge/yamllint) rules.
- [ ] Complies with [Catalog Orgainization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -46,12 +46,12 @@ TEST_YAML_IGNORES=${TEST_YAML_IGNORES:-""}
 
 # Allow ignoring some yaml tests, space separated, should be the basename of the
 # test for example "s2i"
-TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-""}
+TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-"cloudevent"}
```
Comment (vdemeester) [inline test/e2e-tests.sh:49]: Should we had a comment about this one ? (aka depend on `ko`, should use a released image in there — we should also create an issue about it)

Comment (piyush-garg) [reply test/e2e-tests.sh:49]: I am creating an issue for this and also let @iancoffey know :D

I think he is already working, I see a PR #396

Comment (piyush-garg) [reply test/e2e-tests.sh:49]: Created issue https://github.com/tektoncd/catalog/issues/422

**Pull request merged**